### PR TITLE
Update diversity-inclusion.md

### LINKED ISSUE #73: Diversity and Inclusion Page
[Needs content](https://github.com/pythonindia/inpycon2024/blob/main/src/diversity-inclusion.md)

### PR DESCRIPTION
**Pull request by vellankikoti**
Added content for diversity-inclusion page

Fixes #73

### CODE REVIEW

File: src/diversity-inclusion.md [hunk 1]
```diff
@@ -4,3 +4,70 @@ title: Diversity & Inclusion
 ---
 
 Diversity & Inclusion
```
Comment (SaptakS) [inline src/diversity-inclusion.md]: You can remove this line now.

File: src/diversity-inclusion.md [hunk 2]
```diff
@@ -4,3 +4,70 @@ title: Diversity & Inclusion
...
+
+We usually reach out to industry professionals and experts who have delivered multiple talks and workshops at various conferences, covering all the proposal categories mentioned in our Call for Proposals (CFP). These categories include Python in Platform Engineering, Artificial Intelligence and Machine Learning, Python on Hardware, Community, Core Python, Python in Education and Research, Python in Web and Applications, and more. Based on these categories, we identify suitable mentors and schedule sessions to support speakers who need guidance.
+
+If you're interested in becoming a mentor in the future, you can reach out to us by filling out the mentorship form with your experience and details. Our mentorship team reviews these applications and onboards individuals who can contribute to building a supportive learning environment for our community. This process helps ensure that each speaker has the tools and confidence needed to make their presentations a success.
```
Comment (palnabarun) [inline src/diversity-inclusion.md]: This talks about a mentorship form which is not available anymore.

File: src/diversity-inclusion.md [hunk 3]
```diff
@@ -4,3 +4,70 @@ title: Diversity & Inclusion
...
+### Spread the Word
+Help us reach a wider audience by sharing information about PyCon India 2024 and our diversity and inclusion initiatives within your networks.
+
+## Join Us in Building an Inclusive Python Community
+
+We invite you to be a part of PyCon India 2024, where diversity is celebrated, and inclusion is at the heart of our mission. Together, we can create a welcoming and enriching environment where everyone feels empowered to contribute, learn, and grow.
+
+Let's make PyCon India 2024 a memorable and inspiring experience for all. Join us in building a community that truly reflects the diverse world we live in.
+
+For more details about our diversity and inclusion initiatives or to apply for scholarships, visit [PyCon India 2024](https://in.pycon.org/2024/). Let's create a space where every Pythonista can thrive!
```
Comment (palnabarun) [inline src/diversity-inclusion.md]: Scholarships are also closed now.

File: src/diversity-inclusion.md (hunk 1 above)
Comment (vellankikoti) [reply src/diversity-inclusion.md]: removed!

File: src/diversity-inclusion.md (hunk 3 above)
Comment (vellankikoti) [reply src/diversity-inclusion.md]: modified the same.

File: src/diversity-inclusion.md (hunk 2 above)
Comment (vellankikoti) [reply src/diversity-inclusion.md]: modified content for the same

**Comment by anistark** [comment]
Sync with main.
Also add to menu. How do I reach this page?

**Review by anistark — APPROVED** [review]

**Pull request merged**